ui/app/components/modals/reject-transactions/reject-transactions.container.js

```diff
@@ -0,0 +1,18 @@
+import { connect } from 'react-redux'
+import { compose } from 'recompose'
+import RejectTransactionsModal from './reject-transactions.component'
+import withModalProps from '../../../higher-order-components/with-modal-props'
+
+const mapStateToProps = (state) => {
+  const { appState: { modal: { modalState: { props } } } } = state
```

Comment:
These `modalState.props` are available on `ownProps` via `withModalProps`, right?

Reply:
Indeed they are, I'll switch this up